### PR TITLE
Add acceptance tests for checking files sidebar in shared with me page

### PR DESCRIPTION
**Pull request by dpakach**
## Description

add acceptance tests for checking if `shared with me` and `shared-with-others` pages work.

## Related Issue

- Related Issue #1983

## How Has This Been Tested?

:robot: 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5150/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5157/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5158/

<p>





</p></details>

**Comment by individual-it** [comment]
failed with a problem after the login https://drone.owncloud.com/owncloud/phoenix/5158/4/13

I think that was changed in an other PR of yours, does it have some side effect?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5163/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5163/

<p>







</p></details>

File: tests/acceptance/stepDefinitions/sharingContext.js [hunk 1]
```diff
@@ -362,3 +362,7 @@ Then('user {string} should have received a share with these details:', function
 Then('user {string} should have a share with these details:', function (user, expectedDetailsTable) {
   return sharingHelper.assertUserHasShareWithDetails(user, expectedDetailsTable)
 })
+
+Given('the user {string} has created a new public link for resource {string}', function (user, resource) {
+  return shareFileFolder(resource, user, '', 3)
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/sharingContext.js]: can we not use the magic `3` here?

Comment (dpakach) [reply tests/acceptance/stepDefinitions/sharingContext.js]: Yes we can, fixing now.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5173/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5188/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUINotifications</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5188/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5193/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5333/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5335/

<p>




</p></details>

File: tests/acceptance/stepDefinitions/sharingContext.js [hunk 2]
```diff
@@ -141,12 +141,12 @@ Given('the user has shared file/folder {string} with user {string}', function (s
 Given(
   'user {string} has shared file/folder {string} with user {string} with {string} permissions',
   function (sharer, elementToShare, receiver, permissions) {
-    return shareFileFolder(elementToShare, sharer, receiver, SHARE_TYPES.user, permissions)
+    return shareFileFolder(elementToShare, sharer, receiver, sharingHelper.SHARE_TYPES.user, permissions)
```
Comment (skshetry) [inline tests/acceptance/stepDefinitions/sharingContext.js]: Why are you changing this?

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 3]
```diff
@@ -492,3 +494,6 @@ Then('the clipboard content should match permalink of resource {string}', functi
       })
   })
 })
+Then('the thumbnail should be visible in the app-sidebar', function () {
```
Comment (skshetry) [inline tests/acceptance/stepDefinitions/filesContext.js]:
```suggestion

Then('the thumbnail should be visible in the app-sidebar', function () {
```

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5436/

<p>















































</p></details>

**Pull request merged**